### PR TITLE
Add checks for dependencies

### LINKED ISSUE #27: bookman needs warnings about ghostscript and groff
Hi @mvertes,

bookman doesn't work without ghostscript and groff. However, I noticed in Debian that errors shown aren't intuitive to final users. I suggest to test if these programs are present before run bookman.

I will try to write a solution and send a PR today.

Thanks a lot for your very nice work.

Regards,

Eriberto

### PR DESCRIPTION
**Pull request by eribertomota**
bookman command depends of groff and ghostscript. However, the in absence
of these dependencies, no intuitive messages are shown to final users. This
commit adds checks for groff and ghostscript presence and show specific
messages to users.

Closes #27

### CODE REVIEW

File: bookman [hunk 1]
```diff
@@ -85,6 +85,17 @@ do
 done
 shift $(($OPTIND - 1))
 
+# Check for dependencies
+{ echo ok | groff -ms ; } > /dev/null 2> /dev/null || {
```
Comment (mvertes) [inline bookman]:
```suggestion
 groff -ms /dev/null || {
```

File: bookman [hunk 2]
```diff
@@ -85,6 +85,17 @@ do
...
+			printf "ERROR: You need install groff.\n" >&2
+			exit 1
+			}
+
+{ ps2pdf 2>&1 | grep Usage ; } > /dev/null 2> /dev/null || {
```
Comment (mvertes) [inline bookman]:
```suggestion
ps2pdf 2>&1 | grep -q Usage || {
```

**Review by mvertes — CHANGES_REQUESTED** [review]
Ok with the suggested simplifications

**Comment by eribertomota** [comment]
After the changes, can you release a new version? If yes, I will upload it to Debian today.

Thanks for yor efforts!

Eriberto

**Review by mvertes — APPROVED** [review]

**Pull request merged**